fixtures/k8s/kubernetes_resource.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: canaries.flanksource.com/v1
+kind: Canary
+metadata:
+  name: nginx-pod-creation-tests
+  namespace: default
+  labels:
+    "Expected-Fail": "false"
+spec:
+  schedule: '@every 5m'
+  kubernetesResource:
+    - name: ingress-accessibility-check 
+      namespace: default
+      description: "deploy an nginx pod"
+      waitForReady: true 
+      timeout: 10m
+      resources:
+        - apiVersion: v1
+          kind: Pod
+          metadata:
+            name: nginx-pod
+            namespace: default
+          spec:
+            containers:
```

Comment:
Need to add a static ingress and svc + the http check (using the internal nginx service and a host header to match so the dns doesn't need to be setup)